include/formatting/formatting.hpp

```diff
@@ -408,37 +408,37 @@ namespace formatting
 		formatting::internal::ValueWrapperImplementationBase* implementation_;
 	};
 
-	std::string format(const std::string& fmt, 
+    inline std::string format(const std::string& fmt,
```

Comment:
It looks like it is <Tab>inline so these methods indentation is not consistent with other code.